lib/jormungandr/src/Cardano/Wallet/Jormungandr/Binary.hs

```diff
@@ -230,6 +272,7 @@ getTokenTransfer = do
         let _discrimination = discriminationValue headerByte
         case kind of
             -- Single Address
+            -- TODO(akegalj) shouldn't this be `either fail id . decodeAddress (Proxy Jormungandr) <$> getByteString 32` ?
             0x3 -> Address <$> getByteString 32
```

Comment:
Same cyclic import as here https://github.com/input-output-hk/cardano-wallet/pull/362/files#r290338022

Reply:
this shouldn't be relevant as address should be bytestring (I guess address payload only)